package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "snowflake-sdk",
-  "version": "1.6.3",
+  "version": "1.6.4",
```

Comment:
The snowflake team will update the version on the release PR so it's ok to keep this as is. Please revert the version, then we can approve it, thanks

Reply:
Hey @SimbaGithub, reverted the version as requested. Could you have a look?